src/styles/main.css

```diff
@@ -0,0 +1,79 @@
+
+
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.stopwatch {
+  position: relative;
+  border: 1vmin #000;
```

Comment:
The `border` property should include both width and style, e.g., `border: 1vmin dotted #000;` to ensure proper rendering.